extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/user/ModeledUserContext.java

```diff
@@ -137,7 +137,7 @@ public void init(ModeledAuthenticatedUser currentUser) {
         userRecord = new ActivityRecordModel();
         userRecord.setUsername(currentUser.getIdentifier());
         userRecord.setStartDate(new Date());
-        userRecord.setRemoteHost(currentUser.getCredentials().getRemoteHostname());
+        userRecord.setRemoteHost(currentUser.getCredentials().getRemoteAddress());
```

Comment:
The main driver behind this JIRA issue was to gain consistency behind which host information was used in user login history, connection history, active connections, and log messages.  The `getRemoteHost()` method of `RemoteAuthenticatedUser`, used for outputting to logs and for active connections (and thus connection history records) would first look at `X-Forwarded-For` and, if that wasn't present, revert to the `getRemoteAddr()` method on the `HttpServletRequest` class.  As you note above, the user login record was tracked by `getRemoteHostname()` from the `Credentials` object.

In trying to figure out the most reliable way to make this consistent I opted for IP address - it's used already by everything except user login tracking, and, in my anecdotal experience, getting the IP address from the request seems to be more reliable than retrieving the hostname - seems like you often end up with the IP address from the `getRemoteHost()` call in the `HttpServletRequest`, anyway, or something like "localhost."

All that said, I'm not particularly attached to one method or the other - I'd just like to make it consistent.  So, I'm happy to switch all of it over to `getRemoteHostname()` instead of `getRemoteAddress()` if you feel like that's more useful/desired behavior, or we can leave it like this.